### DISCUSSION TITLE
see waypoint ("Zwischenpunkte") in routing-list ("step by step")

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13510] [Ideas]
I think it would be useful to see all waypoint in the detailed "routing-list"

Maybe like this: 
![pic](https://user-images.githubusercontent.com/62500258/148737243-20879b9f-5d45-4edc-af9b-a2948b212c7c.png)

**Comment by pebogufi** [comment]
See also point 4 in
https://github.com/osmandapp/OsmAnd/issues/6742

**Reply by ThomasMeister123** [reply]
March 2019 ;'(

**Reply by vshcherb** [reply]
Yes, we have another issue open to display intermediate points in the list! It's also not yet fixed unfortunately requires quite a big refactoring cause this information is not stored in the list, so it works by proximity....

**Reply by ThomasMeister123** [reply]
Still not available in newest version? 😒
